server/controllers/items/lib/get_authorized_items.js

```diff
@@ -1,40 +1,35 @@
 const __ = require('config').universalPath
 const _ = __.require('builders', 'utils')
-
 const getByAccessLevel = require('./get_by_access_level')
-const { areFriendsOrGroupCoMembers } = __.require('controllers', 'user/lib/relations_status')
 const items_ = __.require('controllers', 'items/lib/items')
 const groups_ = __.require('controllers', 'groups/lib/groups')
 const buildKeysFromShelf = __.require('controllers', 'items/lib/build_keys_from_shelf')
+const getInventoryAccessLevel = require('./get_inventory_access_level')
```

Comment:
[renaming comment warning]
`areFriendsOrGroupCoMembers` or `relations_status` seems to me more explicit than `getInventoryAccessLevel`. I understand the reusage of "access level", but "inventory", to me, is primarely understood as "the user items list". What about `all_access_level` and/or a more explicit variable name (like it was) : `friendsAndGroupCoMembersAccessLevel`

Reply:
in light of the recent discussion on roles:
- the items set their required access level
- the social network (friends and groups) correspond to dynamic roles
- we need to get a user access level to a given inventory to know which items to show

`getUserAccessLevelToInventory`?